src/main/java/dev/latvian/mods/kubejs/client/KeybindRegistryKubeEvent.java

```diff
@@ -0,0 +1,74 @@
+package dev.latvian.mods.kubejs.client;
+
+import com.mojang.blaze3d.platform.InputConstants;
+import dev.latvian.mods.kubejs.util.ClassWrapper;
+import dev.latvian.mods.rhino.util.HideFromJS;
+import net.minecraft.client.KeyMapping;
+import net.neoforged.neoforge.client.settings.KeyConflictContext;
+import net.neoforged.neoforge.client.settings.KeyModifier;
+import org.lwjgl.glfw.GLFW;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class KeybindRegistryKubeEvent implements ClientKubeEvent {
+	private final List<Builder> builders = new ArrayList<>();
+
+	public ClassWrapper<GLFW> getGLFW() {
+		return new ClassWrapper<>(GLFW.class);
+	}
+
+	public Builder register(String keybindId) {
+		Builder builder = new Builder(keybindId);
+		builders.add(builder);
+		return builder;
+	}
+
+	@HideFromJS
+	public List<KubeJSKeybinds.KubeKeybind> build() {
+		return builders.stream().map(Builder::create).toList();
+	}
+
+	public static class Builder {
+		private final String id;
+		private KeyConflictContext keyConflictContext = KeyConflictContext.UNIVERSAL;
+		private KeyModifier keyModifier = KeyModifier.NONE;
+		private InputConstants.Type inputType = InputConstants.Type.KEYSYM;
+		private int keyId = 0;
```

Comment:
The default should be unset (Not Bound, in the vanilla UI), rather than Left Click. (`@Nullable Integer`)